src/domain/community/user/userCard/UserCard.tsx

```diff
@@ -42,7 +42,7 @@ const UserCard = ({
   url,
   roleName,
   loading,
-  isContactable = true,
+  isContactable = false,
```

Comment:
we may miss some envelop icons in places where they should be, 
but at least we don't show anymore envelop icons where they shouldn't be